documentation/source/users/rmg/input.rst

```diff
@@ -450,7 +450,62 @@ or :ref:`pruning <pruning>` can be turned on to speed up your simulation at a sl
 
 .. _filterReactions:
 
-Advanced Setting: Speed Up by Filtering Reactions
+Advanced Setting:  Branching Criterion
+----------------------------------------
+The flux criterion works very well for identifying new species that have high flux
+and therefore pathways that have high throughput flux. However, there are many important
+low flux pathways particularly those that result in net production of radicals such as chain branching reactions in combustion.
+Picking up these pathways with the flux criterion is difficult and almost always requires an appreciably lower
+flux tolerance that would otherwise be necessary causing RMG to pickup many unimportant species. The
+Branching criterion identifies important edge reactions based on the impact they are expected to
+have on the concentrations of core species. The branching specific tolerances shown in the example below
+are the recommended tolerances. These tolerances are much less system dependent than the flux tolerance and
+the recommended values are usually sufficient.
+
+For example ::
+
+	model(
+		toleranceMoveToCore=0.1,
+		toleranceBranchReactionToCore=0.001,
```

Comment:
Could you fix the indentation here?

Reply:
I've made all the indentation consistent with the original pruning indentation. It looks significantly different in the text editor than it does on github.